docs-content/index.html.md.erb

```diff
@@ -13,27 +13,23 @@ owner: Partners
      }
 </style>
 
-<p class="note warning">
-<strong>IMPORTANT: </strong>
-PagerDuty Service Broker for Pivotal Cloud Foundry (PCF) tile is currently in beta and is intended for evaluation and test purposes only.
-Do not use this product in a PCF production environment.
-</p>
-
-This documentation describes PagerDuty Service Broker for Pivotal Cloud Foundry (PCF). PagerDuty Service Broker for PCF enables developers to integrate their PCF apps with PagerDuty's incident management platform.
```

Comment:
We need to keep the [Beta] in the title since we'll still technically be in Beta when we submit

Reply:
Fixed